packages/suite/src/middlewares/suite/analyticsMiddleware.ts

```diff
@@ -68,7 +70,13 @@ const analytics = (api: MiddlewareAPI<Dispatch, AppState>) => (next: Dispatch) =
             );
             break;
         case DEVICE.CONNECT: {
-            const { features, mode } = action.payload;
+            const { features, mode, id } = action.payload;
+
+            // Find a device with the ID that is equal to the ID in action.payload
+            // I can't use action.payload in isBitcoinOnly() because it has a different type (Device). isBitcoinOnly() expects TrezorDevice
+            const connectedDevice = state.devices.find(device => device.id === id);
```

Comment:
better approach will be to add Device type (from connect) to `isBitcoinOnly` util
fields required by this util (features) are the same in both types (TrezorDevice | Device)
then this search will not be needed at all

Reply:
Yes.If that util  may work with multiple shapes of params, just make its params union :+1:

Reply:
btw there was a requirement to log "how many devices are being used at the same time". And maybe this would be a place to do it - whenever user connects a device, also log number of physical devices that are connected along with it. but it could be done later of course